atc/exec/task_step.go

```diff
@@ -339,7 +342,7 @@ func (step *TaskStep) imageSpec(logger lager.Logger, repository *artifact.Reposi
 	} else if config.ImageResource != nil {
 		imageSpec.ImageResource = &worker.ImageResource{
 			Type:    config.ImageResource.Type,
-			Source:  creds.NewSource(boshtemplate.StaticVariables{}, config.ImageResource.Source),
```

Comment:
Ha, guessing this is 'cause it really just never supported interpolation here but the type demanded it. I guess this is a small win.